crates/wasmtime/src/store.rs

```diff
@@ -131,7 +134,25 @@ impl Hash for HostInfoKey {
 
 impl Store {
     /// Creates a new store to be associated with the given [`Engine`].
-    pub fn new(engine: &Engine) -> Store {
+    pub fn new(engine: &Engine) -> Self {
+        Self::new_(engine, None)
+    }
+
+    /// Creates a new store to be associated with the given [`Engine`] and using the supplied
+    /// resource limiter.
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// # use wasmtime::{Engine, Store, StoreLimitsBuilder};
+    /// let engine = Engine::default();
+    /// let store = Store::new_with_limits(&engine, StoreLimitsBuilder::new().instances(10).build());
+    /// ```
```

Comment:
Could the documentation here link to the `ResourceLimiter` trait which discussed which limits can be configured?

Reply:
Oh also, could this link to `StoreLimits` and `StoreLimitsBuilder` for a built-in way to limit resources?